trigger/task.go

```diff
@@ -326,6 +326,10 @@ func (t *taskTriggers) makeData(sub *event.Subscription, pastTenseOverride, test
 
 func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, testNames string) (*notification.Notification, error) {
 	var payload interface{}
+	// We avoid creating BFG ticket in the case that the task is stranded to reduce noise for the Build Baron
```

Comment:
Oops sorry I didn't notice this before: we should only do this for the JiraIssueSubscriberType. It's possible that people put individual slack notifications on tasks finishing or etc and we should still honor that.